### PR TITLE
fix-#1437 Color of Application Components Changed

### LINKED ISSUE #1437: Improper color of Application components
**Summary:**

Certain components of the application make use of the default color `accent` provided by the Android Studio. This affects the uniformity of the application color-scheme.

**Steps to reproduce:**

* Go to Apply for loan and click on data picker. Observe the color of the date picker.
WhatsApp Image 2020-03-21 at 10 44 51 PM(2)

* Go to Accounts - observe the floating action bar color
WhatsApp Image 2020-03-21 at 10 48 04 PM(1)

* Observe the color of OK button on the snackbar
WhatsApp Image 2020-03-21 at 10 48 04 PM

**Expected behaviour:**

All the above-mentioned components should have their color set as the primary color of the application

### PR DESCRIPTION
**Pull request by rutvik-panchal**
Color of Application Components like FloatingActionButton, SnackBar and DatePicker all changed.

Fixes #1437 

Screenshots :

CollageMaker_20200322_200844074

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Review by ShivangiSingh17 — APPROVED** [review]

**Comment by rutvik-panchal** [comment]
@miPlodder Updated the PR for kotlin migration.

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -59,12 +59,19 @@
...
-        <item name="colorAccent">@color/accent</item>
+        <item name="colorAccent">@color/primary</item>
     </style>
     
     <!--Theme for Spash-->
     <style name="SplashTheme" parent="Theme.AppCompat.NoActionBar">
         <item name="android:windowBackground">@drawable/splash_background</item>
     </style>
 
+    <style name="FloatingActionButton" parent="Widget.MaterialComponents.FloatingActionButton">
```
Comment (miPlodder) [inline app/src/main/res/values/styles.xml]: can we use a better name for the `style`, is same as the view. which creates confusion

**Review by miPlodder — CHANGES_REQUESTED** [review]

**Comment by miPlodder** [comment]
can we remove hyphen from the commit-msg also.

File: app/src/main/res/values/styles.xml (hunk 1 above)
Comment (rutvik-panchal) [reply app/src/main/res/values/styles.xml]: Can i rename it to `primaryColorFAB`?

**Comment by rutvik-panchal** [comment]
> can we remove hyphen from the commit-msg also.

@miPlodder Done.

File: app/src/main/res/values/styles.xml (hunk 1 above)
Comment (miPlodder) [reply app/src/main/res/values/styles.xml]: `FloatingActionButtonTheme` should do. which intuitively means Default theme for FAB

Comment (rutvik-panchal) [reply app/src/main/res/values/styles.xml]: @miPlodder Done👍

**Review by miPlodder — APPROVED** [review]
Keep up the good work

**Pull request merged**